.gitignore

```diff
@@ -60,6 +60,7 @@ Lib/test/data/*
 !Lib/test/data/README
 /Makefile
 /Makefile.pre
+Misc/m4/
```

Comment:
Who creates it?

Reply:
I removed the `m4` directories, since I can't "reproduce the bug" anymore :-)
PS: I don't know why I added the '/m4' dir. Maybe its content changed before, but now it's unchanged.